source/data-modeling/field-behaviors.txt

```diff
@@ -0,0 +1,292 @@
+.. _mongoid-field-behaviors:
+
+=========================
+Customize Field Behaviors
+=========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: customize, attributes, optimize, model, configure, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to customize the behavior of fields in {+odm+} models.
+
+Specify Default Values
+----------------------
+
+You can configure fields to have default values by using the ``default`` option.
+Default field values can be either fixed or ``Proc`` values.
+
+The following example specifies a fixed default value for the ``fulfill_by``
+field:
```

Comment:
In this code example, the field is `state` should this be updated or the should the code snippet be updated:
```suggestion
The following example specifies a fixed default value for the ``state``
field:
```

Reply:
Good catch, it should be `state`